### ISSUE TITLE
Cookie modal should be accessible

### ISSUE THREAD

**Issue by DTM-TYPO3** [mindshape-GmbH/mindshape_cookie_consent#55]
In connection with accessibility checks, the requirement that the modal is not in the focus and is therefore difficult to reach using the keyboard keeps cropping up. 

A quick solution ist to put the modal at the beginning of the content - not in the footer, so that the modal links are better accessible for keyboard users. But there should be a better solution coming with the extension itself.

The following should therefore be implemented for the modal:
1. when the dialogue is called up, the focus must be set to the beginning of the dialogue using JavaScript.
2. as long as the dialogue is visible, the focus may only navigate within this dialogue. When leaving the last interactive element with the tab key, the focus must be set to the beginning of the dialogue again.
3. when closing the dialogue, the focus must be set back to the calling position. (For cookie dialogues, this should be the start of the page because users have only just arrived on the page).
4. in order to keep the focus of screen readers (arrow key navigation) in the dialogue, the attributes aria-modal=‘true’ and role=‘dialogue’ must be contained in the topmost element of the dialogue in the HTML hierarchy. 

Greetings
Stefanie

**Comment by featdd** [comment]
Hi @DTM-TYPO3,

this is solved here: c77fdfc1d2bc056b75066ddc2e6e759a57508052
I also just made a new TER release "4.1.0"

Thanks for your hint and have fun with the extension.

Greetings
Daniel
